hosting-companies.md

```diff
@@ -41,6 +41,7 @@ In alphabetical order:
 * [TVC.Net](http://TVC.Net/)
 * [WPEngine.com](http://wpengine.com)
 * [XEL](https://xel.nl)
+* [WPHosting](https://wp-hosting.cz)
```

Comment:
The list is alphabetically sorted, so `WPHosting` should come before `XEL` here.